### PR TITLE
Features/continuous-time examples + update of the documentation.

### PR DESCRIPTION
**Pull request by CMoucer**

### CODE REVIEW

File: PEPit/examples/continuous_time_models/gradient_flow_convex.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0.
+
+        After integrating between :math:`0` and :math:`t`,
+
+        .. math:: f(X_t) - f_\\star \\leqslant \\frac{1}{2t}\\|x_0 - x_\\star\\|^2.
+
+
+    **References**:
+
+    `[1] W. Su, S. Boyd, E. J. Candès (2016).
```
Comment (AdrienTaylor) [inline PEPit/examples/continuous_time_models/gradient_flow_convex.py:42]: mb add ref to your own work, which explains how to do this analysis with PEPs?

File: PEPit/examples/continuous_time_models/gradient_flow_strongly_convex.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+    **Theoretical guarantee**:
+
+        The following **tight** guarantee can be found in [1, Proposition 11]:
+
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t) \\leqslant -2\\mu\\mathcal{V}(X_t).
+
+    **References**:
+
+    `[1] D. Scieur, V. Roulet, F. Bach and A. D'Aspremont (2017).
```
Comment (AdrienTaylor) [inline PEPit/examples/continuous_time_models/gradient_flow_strongly_convex.py:38]: same: ref to your work?

File: PEPit/examples/continuous_time_models/gradient_flow_convex.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    with :math:`X_{0}:= x_0 \\in \\mathbf{R}^d`.
+
+    **Theoretical guarantee**:
+
+        The following **tight** guarantee can be found in [1, p. 7]:
+
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0.
+
+        After integrating between :math:`0` and :math:`t`,
```
Comment (AdrienTaylor) [inline PEPit/examples/continuous_time_models/gradient_flow_convex.py:35]: integrate between 0 and T (capital) ? to differentiate between t and T?
minor, though!

File: PEPit/examples/continuous_time_models/accelerated_gradient_flow_convex.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0.
+
+        After integrating between :math:`0` and :math:`t`,
+
+        .. math:: f(X_t) - f_\\star \\leqslant \\frac{2}{t^2}\\|x_0 - x_\\star\\|^2.
+
+    **References**:
+
+    `[1] W. Su, S. Boyd, E. J. Candès (2016).
```
Comment (AdrienTaylor) [inline PEPit/examples/continuous_time_models/accelerated_gradient_flow_convex.py:42]: add ref to your own work for the description of the PEP formulation for this setup?

File: PEPit/examples/continuous_time_models/accelerated_gradient_flow_convex.py [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+    with :math:`X_{0}:= x_0 \\in \\mathbf{R}^d`.
+
+    **Theoretical guarantee**:
+
+        The following **tight** guarantee can be verified in [1, Section 2]:
+
+        .. math:: \\frac{d}{dt}\\mathcal{V}(X_t, t) \\leqslant 0.
+
+        After integrating between :math:`0` and :math:`t`,
```
Comment (AdrienTaylor) [inline PEPit/examples/continuous_time_models/accelerated_gradient_flow_convex.py:36]: very minor: integrate between 0 and T (and then guarantee in terms of T)?

**Review by AdrienTaylor — APPROVED** [review]
I would just do a few minor modifications in the comments!

**Review by bgoujaud — APPROVED** [review]
All good to me. I just made a very few number of convention uniformizations.

Tests are running good and documentation is nice!

**Review by AdrienTaylor — APPROVED** [review]

**Pull request merged**